package.json

```diff
@@ -4,16 +4,18 @@
   "description": "bugsnag logger for book",
   "main": "index.js",
   "dependencies": {
-    "bugsnag": "1.6.3",
+    "bugsnag": "1.7.0",
     "nock": "0.36.2",
     "xtend": "3.0.0"
   },
   "devDependencies": {
+    "book": "1.3.1",
     "mocha": "1.20.1",
-    "book": "1.3.1"
+    "nsp": "^2.6.1"
```

Comment:
Valid. Thought it might be useful to have for future scans but should prob keep as a global tool and not bloat the package. Updated.

nsp just checks if there are know CVEs in your dependencies.